lib/tasks/argo.rake

```diff
@@ -17,14 +17,17 @@ end
 task :default => [:ci]
 
 task :ci do
-  ENV['RAILS_ENV'] = 'test'
-  Rake::Task['argo:install'].invoke
-  jetty_params = jettywrapper_load_config()
-  error = Jettywrapper.wrap(jetty_params) do
-    Rake::Task['argo:repo:load'].invoke  # load 'em all!
-    Rake::Task['spec'].invoke
+  if Rails.env.test?
```

Comment:
why not make this:

```
fail unless Rails.env.test?
```
and then change `.travis.yml` to always set the RAILS_ENV=test? That way you cannot run the `ci` task without explicitly setting RAILS_ENV=test. That seems more consistent to me.

See http://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-.travis.yml